src/einsteinpy/symbolic/tensor.py

```diff
@@ -31,8 +32,29 @@ def __init__(self, arr):
             raise TypeError("Only multi-dimensional list or Sympy Array is expected")
 
     def __getitem__(self, index):
+
         return self.arr[index]
```

Comment:
Why unnecessary newline here?

Reply:
I was going to write a docstring, but then I thought that in that case it was not neccesary, but I've forgotten to delete the newline.